encrypted-media-respec.html

```diff
@@ -680,36 +591,33 @@ <h3><dfn>Navigator</dfn> Extension: <code>requestMediaKeySystemAccess()</code></
                 Implementations MUST meet all related requirements and SHOULD follow related recommendations such that the risks on in an secure context would be similar.
               </p>
             </div>
-
+            <p>When this method is invoked, the user agent must run the following steps:</p>
```

Comment:
The rest of the spec uses capitalized "MUST" to signal RFC2119 terms.

```suggestion
            <p>When this method is invoked, the user agent MUST run the following steps:</p>
```

To be replaced in all the occurrences that this PR introduces.